.cirrus.yml

```
@@ -0,0 +1,67 @@
+run_tests: &RUN_TESTS
+  install_cibuildwheel_script:
+    - python -m pip install -e ".[dev]" pytest-custom-exit-code
+  run_cibuildwheel_tests_script:
+    - python ./bin/run_tests.py
+
+
+linux_x86_task:
+  compute_engine_instance:
+    image_project: cirrus-images
+    image: family/docker-builder
+    platform: linux
+    cpu: 8
+    memory: 8G
+
+  install_pre_requirements_script:
+    - apt install -y python3-venv python-is-python3
+  <<: *RUN_TESTS
+
+linux_arm64_task:
+  arm_container:
+    image: python:3.10
+    cpu: 1
+    memory: 1G
+    additional_containers:
+      - name: dockerdaemon
+        privileged: true
+        cpu: 3
+        memory: 3G
+        image: docker:dind
+        command: dockerd
+        port: 2375
+        env:
+          DOCKER_DRIVER: overlay2
+          DOCKER_TLS_CERTDIR: ""
+  env:
+    DOCKER_HOST: tcp://localhost:2375
+
+  install_pre_requirements_script:
+    - curl -sSL https://get.docker.com/ | sh
+  <<: *RUN_TESTS
+
+windows_x86_task:
+  # The task takes ~55 minutes while the timeout happens
+  # after 60 minutes by default, let's allow some wiggle room.
+  timeout_in: 70m
```

Experience tells me a bit extra leeway would be good

```suggestion
  timeout_in: 90m
```